lib/collection/response.js

```diff
@@ -215,8 +223,8 @@ _.extend(Response.prototype, /** @lends Response.prototype */ {
         }
 
         // we create the body string first from stream and then fallback to body
-        return 'data:' + mime._sanitisedContentType + ';base64, ' + (((this.stream != null) &&
-            util.bufferOrArrayBufferToBase64(this.stream)) || ((this.body != null) && util.btoa(this.body)) || E);
```

Comment:
Eslint recommendation to not use !=. I always felt that != null should be allowed. 😕